docs/website/docs/dlt-ecosystem/destinations/weaviate.md

```diff
@@ -244,6 +244,17 @@ Reserved property names like `id` or `additional` are prefixed with underscores
 - `vectorizer`: (str) the name of [the vectorizer](https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules) to use. The default is `text2vec-openai`.
 - `moduleConfig`: (dict) configurations of various Weaviate modules
 
+### Configure Weaviate modules
+
+The default configuration for the Weaviate destination uses `text2vec-openai`.
+To configure another vectorizer or a generative module, replace the default `module_config` value by updating `config.toml`:
+
+```toml
+[destination.weaviate]
+module_config={text2vec-openai = {}, generative-openai = {}}
```

Comment:
it works? we used 
```
[destination.weaviate.module_config."text2vec-openai"]
model = "ada"
modelVersion = "002"
type = "text"
[destination.weaviate.module_config."generative-openai"]
model = "gpt-3.5-turbo"
```

Reply:
@AstrakhantsevaAA Yes, this is another way of having nested data in toml.

Reply:
cool!